tools/add_triage_signature.py

```diff
@@ -2399,6 +2399,61 @@ def _process_ticket(self, url, issue_key):
             )
 
 
+class ControllerFailedToStart(ErrorSignature):
+    """
+    This signature looks for missing pivot URL in rpm-ostree status of the control-plane hosts
+    """
+
+    PODS_PATH = "*_bootstrap_*.tar.gz/logs_host_*/log-bundle-*.tar.gz/log-bundle-*/resources/pods.json"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(
+            *args,
+            **kwargs,
+            function_impact_label="assisted_controller_failed_to_start",
+            comment_identifying_string="h1. Assisted Installer Controller failed to start",
+        )
+
+    def _process_ticket(self, url, issue_key):
+        md = get_metadata_json(url)
+        cluster_hosts = md["cluster"]["hosts"]
+        bootstrap_node = [host for host in cluster_hosts if host["bootstrap"]][0]
+
+        if bootstrap_node["progress"]["current_stage"] != "Waiting for controller":
+            return
+
+        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=get_metadata_json(url)["cluster"]["id"])
```

Comment:
Don't download metadata twice

```suggestion
        cluster = get_metadata_json(url)["cluster"]
        bootstrap_node = [host for host in cluster["hosts"] if host["bootstrap"]][0]

        if bootstrap_node["progress"]["current_stage"] != "Waiting for controller":
            return

        triage_logs_tar = get_triage_logs_tar(triage_url=url, cluster_id=cluster["id"])
```